src/Angor/Client/Models/FlattenedProjectTest.cs

```diff
@@ -0,0 +1,88 @@
+using Angor.Shared.Models;
+
+namespace Angor.Client.Models;
+
+public class FlattenedProjectTest
```

Comment:
What's this a test class?

Reply:
Testing flattening the root object, not in use for now